github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -104,6 +104,7 @@ presubmits:
             - "/bin/bash"
             - "-c"
             - "cd images && ./publish_image.sh -b bootstrap docker.io kubevirtci"
+            - "cd images && ./publish_image.sh -b golang docker.io kubevirtci"
```

Comment:
note for the future: These lines need to go into a all-in-one release script, since otherwise we can't test on PRs.